source/fundamentals/crud/read-operations/geo.txt

```diff
@@ -93,7 +93,7 @@ the field on the collection. The following snippet creates an index on the
    db.shipwrecks({coordinates: "2d"});
 
 See the
-:manual:`MongoDB server manual page on legacy coordinate pairs </geospatial-queries/#legacy-coordinate-pairs>`
+:manual:`MongoDB Server manual page on legacy coordinate pairs </geospatial-queries/#legacy-coordinate-pairs>`
```

Comment:
Suggestion:
For all instances of "MongoDB Server", I think it could be easier to use the source constant:
```
:manual:`{+mdb-server+} manual page...
```